commit e552bff1de0948f1356cf3b2e659715f1cd7a426 (refs/changes/86/3886/3)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2018-05-02T09:07:52-07:00 (1 year, 5 months ago)
    
    TRITON-354 vm-agent should use node v6.x
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/Makefile b/Makefile
index 9eafbc2..bcefcfb 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2016, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 #
@@ -33,7 +33,7 @@ JSSTYLE_FLAGS =
 
 # Should be the same version as the platform's /usr/node/bin/node.
 NODE_PREBUILT_TAG =	gz
-NODE_PREBUILT_VERSION =	v4.6.1
+NODE_PREBUILT_VERSION =	v6.14.0
 ifeq ($(shell uname -s),SunOS)
 	# Allow building on a SmartOS image other than sdc-smartos/1.6.3.
 	NODE_PREBUILT_IMAGE =	18b094b0-eb01-11e5-80c1-175dac7ddf02
diff --git a/package.json b/package.json
index 8cd457f..913aaef 100644
--- a/package.json
+++ b/package.json
@@ -1,12 +1,12 @@
 {
     "name": "vm-agent",
     "description": "SmartDC Node VM Agent",
-    "version": "1.7.0",
+    "version": "1.7.1",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
         "assert-plus": "0.2.0",
-        "bunyan": "1.5.1",
+        "bunyan": "1.8.12",
         "cueball": "2.1.1",
         "deep-diff": "0.3.3",
         "eslint": "2.13.1",
